transport/encode_decode.go

```diff
@@ -208,13 +230,19 @@ func decodeGetEvaluationsRequest(c echo.Context) (interface{}, error) {
 // requests into a domain.EvaluationsByFeatureRequest that can be passed to the ProxyService
 func decodeGetEvaluationsByFeatureRequest(c echo.Context) (interface{}, error) {
 	envID := c.Param("environment_uuid")
-	target := c.Param("target")
+	targetIdentifier := c.Param("target")
 	feature := c.Param("feature")
 
+	target, err := extractTarget(c)
+	if err != nil {
+		return nil, errBadRequest
```

Comment:
What's the behaviour on SaaS for this scenario? Just wondering if we should be more defensive and log an error then fall back to loading from redis if a malformed target comes through or return an error like here

Reply:
Good question, I'll have a look and see what we do there

Reply:
Yea it looks like in Saas we do what you're calling out and if we get an error decoding the target from the header we log it and then do a lookup in redis. I'll update this to do the same.